src/scripts/hubot-jenkins-notifier.coffee

```diff
@@ -91,17 +100,31 @@ class JenkinsNotifier
     return req.body
 
   process: (req,res) ->
+    @logMessage(undefined, "jenkins-notifier: Incoming request at #{req.url}")
+
     query = querystring.parse(url.parse(req.url).query)
 
     res.end('')
 
     envelope = {notstrat:"Fs"}
     envelope.user = {}
-    envelope.user.room = envelope.room = query.room if query.room
+    
     envelope.notstrat = query.notstrat if query.notstrat
     envelope.notstrat = 'FS' if query.always_notify #legacy
     envelope.user.type = query.type if query.type
 
+    if query.user && query.room
+      @logMessage(query.trace, "Cannot use room and user together")
```

Comment:
This doesn't make sense to be wrapped in logMessage. Its always an error, so always echo out the error.

I'm thinking it'd be worth also outputting both query.user and query.room so it can be debugged later.